packages/terra-table/src/Table.jsx

```diff
@@ -104,6 +104,11 @@ const propTypes = {
    */
   overflowColumns: PropTypes.arrayOf(columnShape),
 
+  /**
+   * Table body columns information.
+   */
+  tableBodyColumns: PropTypes.arrayOf(columnShape),
```

Comment:
The Terra Table component should not have two different ways to accept its column data.  We should not make a change like this as it will be difficult to maintain.

Reply:
Updated.